odoo/service/db.py

```diff
@@ -69,12 +69,10 @@ def _initialize_db(id, db_name, demo, lang, user_password, login='admin', countr
                 modules._update_translations(lang)
 
             if country_code:
-                countries = env['res.country'].search([('code', 'ilike', country_code)])
-                if countries:
-                    comp_local = {'country_id': countries[0].id}
-                    if countries[0].currency_id:
-                        comp_local['currency_id'] = countries[0].currency_id.id
-                    env['res.company'].browse(1).write(comp_local)
+                country = env['res.country'].search([('code', 'ilike', country_code)])[0]
+                env['res.company'].browse(1).write({'country_id': country_code and country.id, 'currency_id': country_code and country.currency_id.id, 'phone':phone})
+            elif phone:
+                env['res.company'].browse(1).write({'phone': phone})
 
```

Comment:
@antonylesuisse is this ok for you ? I changed because if the user does not choose a country code, the line is never reached and the phone was not stored.